### PR TITLE
SAK-48997 Grader. Use BasicText config for private and feedback editors

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48997

### CODE REVIEW

**Comment by adrianfish** [comment]
Screenshot 2023-06-14 at 09 46 30

**Review by jesusmmp — APPROVED** [review]

File: webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js [hunk 1]
```diff
@@ -259,17 +261,17 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
...
 
-  _replaceWithEditor(id, changedCallback) {
+  _replaceWithEditor(options) {
 
-    const editor = sakai.editor.launch(id, {
+    const editor = sakai.editor.launch(options.id, {
       autosave: { delay: 10000000, messageType: "no" },
       startupFocus: true,
-      toolbarSet: "Basic"
+      toolbarSet: options.fullEditor ? "Full" : "BasicText"
```
Comment (kunaljaykam) [inline webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js:269]: I like it :+1:

File: webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js [hunk 2]
```diff
@@ -286,7 +288,7 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
   _toggleInlineFeedback(e, cancelling) {
 
     if (!this.feedbackTextEditor) {
-      this.feedbackTextEditor = this._replaceWithEditor("grader-feedback-text-editor");
+      this.feedbackTextEditor = this._replaceWithEditor({ id: "grader-feedback-text-editor", fullEditor: true });
```
[the remark this answers is not in the document]
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js:291]: yes, that's where we need it, in the inline feedback area.

**Review by kunaljaykam — APPROVED** [review]
I tested it. Works fine. Thanks @adrianfish

**Pull request merged**